geomstats/information_geometry/exponential.py

```diff
@@ -0,0 +1,167 @@
+import geomstats.backend as gs
+import geomstats.errors
+from geomstats.geometry.base import OpenSet
+from geomstats.geometry.euclidean import Euclidean
+from geomstats.geometry.riemannian_metric import RiemannianMetric
+
+from scipy.stats import expon
+
+
+class ExponentialDistributions(OpenSet):
+    """Class for the manifold of exponential distributions."""
+
+    def __init__(self):
```

Comment:
Add something like "This is the parameter space of exponential distributions, i.e. the half-line of positive reals."